### PR TITLE
Rework

### PR DESCRIPTION
**Pull request by daguag**
I started to rework the document, especially the Readme and the Introduction.
I assume there shouldn't be serious conflicts.

### CODE REVIEW

**Comment by blochberger** [comment]
Do you resolve the merge conflicts?

File: README.md [hunk 1]
```diff
@@ -1,27 +1,24 @@
...
 
+**TODO:** add logo
 
+This document outlines basic aspects of mobile apps which should be considered for writing mobile apps in order to build privacy protection into the apps.
 
+We are convinced that **privacy protection is an attractive feature** and can attract addtional users. We aim on providing **easy to implement** privacy features **without limiting** the functions of your app.
 
+Basic concepts of privacy problems and how to address them using privacy enhancing technologies (PETs) are combined with legal hints how you comply with data protection law, i.e. the new european data protection regulation.
 
+The guideline is work in progress and written as part of the [APPpets](http://app-pets.org/home/) project. Contributions, feedback and discussion is welcomed.
```
Comment (blochberger) [inline README.md]: Should we mention BMBF as well?

**Comment by daguag** [comment]
I wasn't sure if I had the access rights for that. But now I resolved the conflicts.

File: README.md [hunk 2]
```diff
@@ -1,27 +1,24 @@
...
-This document describes how developers can, should, or must protect their apps for more privacy or compliance.
+To get an overview of the developer guidelines document have a look at the [introduction](./Introduction.md)
 
 ## Contribution
 
 ### Places of interest:
 
 * Reading / Link for sharing: [https://blochberger.gitbooks.io/developer-guidelines](https://blochberger.gitbooks.io/developer-guidelines)
 * Issues/Discussions: [https://www.gitbook.com/book/blochberger/developer-guidelines/discussions](https://www.gitbook.com/book/blochberger/developer-guidelines/discussions)
+    * **[move to apppets repository?]**
```
Comment (blochberger) [inline README.md]: I'm for it. Maybe we should do that at our next meeting.

**Pull request merged**